### PR TITLE
add audit webhook logs config

### PR DESCRIPTION
**Pull request by jgsqware**
Signed-off-by: Julien Garcia Gonzalez <julien@giantswarm.io>

### CODE REVIEW

File: platforms/azure/giantnetes/main.tf [hunk 1]
```diff
@@ -84,6 +84,7 @@ locals {
     "ETCDInitialClusterSingle" = "etcd1=https://etcd1.${var.base_domain}:2380"
     "G8SVaultToken"            = "${var.nodes_vault_token}"
     "K8SAPIIP"                 = "${var.k8s_api_ip}"
+    "K8SAUDITWEBHOOKPORT"      = "${var.k8s_audit_webhook_port}"
```
Comment (calvix) [inline platforms/azure/giantnetes/main.tf]: no need for uppercase for whole name :P

File: README.md [hunk 2]
```diff
@@ -24,4 +24,4 @@ ln -sf ${GOPATH}/bin/terraform-provider-gotemplate ${HOME}/.terraform.d/plugins/
 
 Please follow:
 - [installation guide for AWS](docs/installation-guide-aws.md).
-- [installation guide for Azure](docs/installation-guide-azure.md).
+- [installation guide for Azure](docs/installation-guide-azure.md).
```
Comment (tuommaki) [inline README.md]:
```suggestion
- [installation guide for Azure](docs/installation-guide-azure.md).
```
Drop accidental drive-by space :slightly_smiling_face:

**Comment by calvix** [comment]
I will get back on this after i merge 1.14, let see if CI will be green again

**Comment by calvix** [comment]
all green, @jgsqware we can merge i assume

**Comment by jgsqware** [comment]
@T-Kukawka here is the audit PR for the the api-server change. It was stuck because of CI. I will handover this to you.

**Comment by calvix** [comment]
whole CI was green, so no need to rerun just because of readme change

**Comment by jgsqware** [comment]
> whole CI was green, so no need to rerun just because of readme change

was not meant to rerun it, I've just merged the change sorry
